controllers/orders.js

```diff
@@ -0,0 +1,50 @@
+const Order = require('../models/order');
+
+module.exports = {
+    index,
+    show,
+    create,
+    delete: deleteOrder,
+};
+
+async function index(req, res) {
+    try {
+        const orders = await Order.find({}).populate('products');
```

Comment:
this function needs a revision so that the functionality is specific to one buyer. currently, this will show every user's orders

Reply:
function has been revised to: ```const orders = await Order.find({ googleID: req.user.googleId }).populate('products'); ``` 
Let me know if this will work